README.md

```diff
@@ -462,6 +463,28 @@ to live.
 
 Note: `chronos-sync.rb` does not delete jobs by default. You can pass the `--delete-missing` flag to `chronos-sync.rb` to remove jobs. Alternatively, you can manually remove it using the API or web UI.
 
+## Mesos Framework Authentication
+
+To enable the credentials in Chronos:
+
+* Run mesos-master with the credentials flag and the path to file with authorized users + their secrets.
```

Comment:
Also note that the Mesos `--credentials` flag is specific to the built-in CRAM-MD5 authentication mechanism. Other custom authentication modules may have different instructions.